cellprofiler_core/preferences/__init__.py

```diff
@@ -240,12 +247,11 @@ def config_read(key):
             # Fix problems with some 32-bit
             import wx
 
-            entry_type = get_config().GetEntryType(key)
```

Comment:
Might be outside the scope of this PR, but maybe the results of `get_config()` should be set to a variable on its first call and used below? That way the function doesn't have to be called multiple times.

Reply:
I wondered if it was intentionally set up this way so that when running multiple copies of CellProfiler you'd keep using the most up-to-date config settings. I imagine we could probably still cut down on redundant work by fetching and passing in the config when working on multiple keys in quick succession?